includes/wf_crm_webform_postprocess.inc

```diff
@@ -1848,6 +1860,10 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       'financial_type_id' =>  $contributionParams['financial_type_id'],
     );
 
+    if ($paymentType === 'deferred' && !empty($contributionParams['auto_renew'])) {
+      $contributionRecurParams['auto_renew'] = 1;
+    }
+
```

Comment:
The concept of paymentType is deprecated.  From what I can see this is the only bit that stops this working with non "manual" payment processors?  We should not need to differentiate.

Reply:
Can you clarify what do you mean by `deprecated` because here in this  module it operate like a flag to distinguish between live and manual payments, and based on that it totally call different API end point : 

https://github.com/colemanw/webform_civicrm/blob/b89be6c942ff5e621fb7df0085721b27dd9d57f7/includes/wf_crm_webform_postprocess.inc#L1886-L1889

so I am not sure how it is deprecated.

Reply:
@omarabuhussein The `Contribution.transact` API is also deprecated and this whole part of webform needs to be replaced eventually with `Order.create` + `PaymentProcessor.Pay` APIs. I think that `if (!empty($contributionParams['auto_renew'])) {` should be fine here?